src/linters/index.js

```diff
@@ -1,4 +1,5 @@
 const Black = require("./black");
+const RuboCop = require("./rubo-cop");
```

Comment:
Please keep imports and the content of the `linters` object sorted alphabetically :)

Reply:
If only there were some tool to remind me to do this...